rsts/deployment/security/security.rst

```diff
@@ -0,0 +1,46 @@
+.. _security-overview:
+
+###################
+Security Overview
+###################
+
+Here we cover security aspects of running your flyte deployments. In the current state we will be covering the user
+which is used for running the flyte services and we will go through why we do this and not run them as root user
+
+#################
+Why non-root user
+#################
+Up until now all the containers that are used for bringing up the flyte deployment used the default root user to bring
+up the services. We use docker for container packaging and by default its containers run as root. This gives full
```

Comment:
Generally speaking, let's talk about the things as facts... docs are sort of timeless.. people will read them and they don't need to learn about the transition from root to non-root IMHO... they just need to learn about why we are running as non-root... 

```suggestion
Flyte uses docker for container packaging and by default its containers run as root. This gives full
```

Reply:
Sounds good. Done